src/engine/server.h

```diff
@@ -55,6 +55,7 @@ class IServer : public IInterface
 	virtual const char* GetAnnouncementLine(char const* FileName) = 0;
 
 	virtual void SendWebhookMessage(const char *pURL, const char *pMessage, const char *pUsername = "", const char *pAvatarURL = "") = 0;
+	virtual void SendWebhook1vs1(const char *pMessage, const char *pUsername, const char *pAvatarURL) = 0;
```

Comment:
This is not required, you can use `SendWebhookMessage()` for this aswell.

Reply:
Damn, I just realized what a stupid thing I did, it could have been much simpler, well, if you want you can rework my code down to two lines ^^